css/_project-custom.scss

```diff
@@ -1757,39 +1760,39 @@ html {
   @include add-list-reset;
 }
 
-.usa-body-content, .usa-prose {
-  h1,
-  h2,
-  h3 {
-    @include txt-normalize-headings;
-  }
-  h2 { @extend .txt-h2; }
-  h3 {
-    @extend .txt-h3;
-    & + p {
-      @include u-margin-top(1);
-    }
-  }
-  p { @extend .txt-p; }
-  ul { @extend .txt-ul; }
-  li { @extend .txt-li; }
-  p a {
-    @extend .txt-a;
-    &.usa-lede-link {
-      @include u-text($gray-90);
-      @include u-underline($gray-20);
-      &:hover{
-        @include u-underline($gray-80);
-      }
-    }
-  }
-  code { @extend .txt-code; }
-  aside {
-    code {
-      @include u-bg($gray-warm-10);
-    }
-  }
-}
+// .usa-body-content, .usa-prose {
```

Comment:
@thisisdano can you confirm this Sass code is not needed? It didn't seem to be in use and the `.usa-prose` here was adding confusion when I was trying to test the default behavior.

Reply:
👀

Reply:
This is not necessary any longer — I think it was a first draft of mine of what became your work.

Reply:
Removed! is any of the CSS above this starting from `CUSTOM PROJECT SASS` in use too? Starting from https://github.com/uswds/uswds-site/pull/615/files#diff-8ef0ab0a17bcd64e66ebd7c22ee57320R1633 and up to this code.

Reply:
I think `.is-inverse` is the only rule worth keeping in that section